### PR TITLE
AVIATORAPI-90 Delete access from private flavor

### PR DESCRIPTION
**Pull request by lvillasica**

### CODE REVIEW

**Comment by lvillasica** [comment]
@musashi-dev/ronins 
I always get 404 error
```
FAIL aviator/openstack/compute/v2/admin/remove_flavor_from_project leads to a valid response when valid params are provided (0.47s)
  Expected: 200
    Actual: 404
  (eval):8:in `must_equal'
  /home/lvillasica/Documents/apps/aviator/test/aviator/openstack/compute/v2/admin/remove_flavor_from_project_test.rb:117:in `block (2 levels) in <class:Test>'
```

**Comment by lvillasica** [comment]
```
vagrant@devstack:~/devstack$ nova flavor-access-list --flavor 100
+-----------+----------------------------------+
| Flavor_ID | Tenant_ID                        |
+-----------+----------------------------------+
| 100       | 3697be792da2419f99a63e4d0e34c96c |
+-----------+----------------------------------+
```

**Comment by devpopol** [comment]
how about passing in tenant_id?

**Comment by jocaba** [comment]
+1  I'd just like to note that Openstack's API docs are wrong about this one. Martin's POST request was the needed magic ingredient

File: test/aviator/openstack/compute/v2/admin/remove_flavor_from_project_test.rb [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        p[:flavor_id] = flavor_id
+        p[:tenant] = tenant
+      end
+
+      request.url.must_equal url
+    end
+
+
+    validate_response 'valid params are provided' do
+      # must be hardcoded so as not to inadvertently alter random resources
```
Comment (stefanymarie) [inline test/aviator/openstack/compute/v2/admin/remove_flavor_from_project_test.rb]: how about creating the precondition? i.e. create a flavor for project

**Comment by lvillasica** [comment]
Thanks to @stefanymarie :hand:

File: test/aviator/openstack/compute/v2/admin/remove_flavor_from_project_test.rb (hunk 1 above)
Comment (lvillasica) [inline test/aviator/openstack/compute/v2/admin/remove_flavor_from_project_test.rb]: @stefanymarie So we will need create flavor api to be merged first, right?

Comment (choodur) [inline test/aviator/openstack/compute/v2/admin/remove_flavor_from_project_test.rb]: @lvillasica yes, https://github.com/musashi-dev/aviator/pull/52 is now merged.

**Comment by lvillasica** [comment]
@musashi-dev/ronins please review :smiley_cat:

**Comment by stefanymarie** [comment]
+1

**Comment by choodur** [comment]
+1.

**Pull request merged**